cssom-1/Overview.bs

```diff
@@ -911,6 +911,10 @@ interface CSSStyleSheet : StyleSheet {
   [SameObject] readonly attribute CSSRuleList cssRules;
   unsigned long insertRule(CSSOMString rule, optional unsigned long index = 0);
   void deleteRule(unsigned long index);
+
+  [SameObject] readonly attribute CSSRuleList rules;
+  long addRule(optional DOMString selector = "undefined", optional DOMString style = "undefined", optional unsigned long index);
```

Comment:
Wow. I guess in Blink this is done via https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/css/css_style_sheet.idl?q=css_style_sheet.idl&sq=package:chromium&g=0&l=1 which bottoms out at https://chromium.googlesource.com/chromium/src/+/master/third_party/blink/renderer/bindings/IDLExtendedAttributes.md#DefaultValue_p

/cc @yukishiino as Blink bindings team member. I hope we can get rid of [DefaultValue] and prevent any future APIs from ending up this weird.